astroid/arguments.py

```diff
@@ -86,7 +86,11 @@ def has_invalid_keywords(self) -> bool:
         """
         return len(self.keyword_arguments) != len(self._unpacked_kwargs)
 
-    def _unpack_keywords(self, keywords, context: InferenceContext | None = None):
+    def _unpack_keywords(
+        self,
+        keywords: list[tuple[str | None, nodes.NodeNG]],
```

Comment:
I feel like https://github.com/pylint-dev/astroid/commit/b72eaa7d0440701a72e7b79ca9b1f7e8e9603b74 might have typed `arg` incorrectly. Will investigate this myself.

Reply:
Turns out to be correct, weirdly enough..